src/transformers/tape.js

```diff
@@ -78,8 +78,6 @@ const tPropertiesUnsupported = new Set([
     'notDeepLooseEqual',
     'notLooseEqual',
     'notLooseEquals',
-
-    'skip',
```

Comment:
I've rebased against your `master`. I can get the `looseEquals` test to fail if I remove it from the `tPropertiesUnsupported` Set, but not the `skip` property. If I remove `timeoutAfter`, the test for that property still passes.

Do you have any ideas?